internal/usage/usage.go

```diff
@@ -311,4 +311,15 @@ Valid values: KEYS|IAM_ROLE.`
 Valid values: 15|30|60.`
 	SnapshotIntervalHours = `Number of hours between snapshots.
 Valid values: 6|8|12|24.`
+	LiveMigrationDestinationClusterName      = "Human-readable label that identifies the Atlas destination cluster."
+	LiveMigrationDestinationProjectID        = "Human-readable label that identifies the Atlas destination project."
```

Comment:
copy/paste? for project ID is probably not a label (name)

Reply:
Yup, in the spec :) Fixing both